src/test/java/seedu/address/logic/parser/UpdatePhotoCommandParserTest.java

```diff
@@ -0,0 +1,30 @@
+package seedu.address.logic.parser;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.UpdatePhotoCommand;
+import seedu.address.logic.parser.exceptions.ParseException;
+
+public class UpdatePhotoCommandParserTest {
```

Comment:
Just a suggestion, can consider successful parsing of correct input as well for rigorous testing. You can follow more guidelines on what to test [here](https://nus-cs2103-ay2324s1.github.io/website/schedule/week11/topics.html#w11-3-test-cases-combining-multiple-inputs)